sdk/keyvault/azure-security-keyvault-jca/src/main/java/com/azure/security/keyvault/jca/AuthClient.java

```diff
@@ -131,20 +131,20 @@ public String getAccessToken(String resource, String aadAuthenticationUrl,
      * @param identity the user-assigned identity (null if system-assigned).
      * @return the authorization token.
      */
-    private String getAccessTokenOnAppService(String resource, String identity) {
+    private String getAccessTokenOnAppService(String resource, String clientId) {
         LOGGER.entering("AuthClient", "getAccessTokenOnAppService", resource);
         LOGGER.info("Getting access token using managed identity based on MSI_SECRET");
-        if (identity != null) {
-            LOGGER.log(INFO, "Using managed identity with client ID: {0}", identity);
+        if (clientId != null) {
+            LOGGER.log(INFO, "Using managed identity with client ID: {0}", clientId);
         }
         String result = null;
 
         StringBuilder url = new StringBuilder();
         url.append(System.getenv("MSI_ENDPOINT"))
            .append("?api-version=2017-09-01")
            .append(RESOURCE_FRAGMENT).append(resource);
-        if (identity != null) {
-            url.append("&clientid=").append(identity);
+        if (clientId != null) {
+            url.append("&clientid=").append(clientId);
```

Comment:
what if move "LOGGER.log(INFO, "Using managed identity with client ID: {0}", clientId);" here

Reply:
Hi, what is your concern of it? I think it's helpful to inform users about the flow of acquiring tokens and also debug.

Reply:
IMU, @backwind1233 's concern is that `if (clientId != null)` appeared more than one time.

Reply:
@chenrujun  yes 
@yiliuTo 
![image](https://user-images.githubusercontent.com/4465723/113372867-2d5a3580-939c-11eb-8e62-f5f35011b7a5.png)